classes/abstracts/ActionScheduler.php

```diff
@@ -149,7 +173,8 @@ public static function init( $plugin_file ) {
 
 		// Ensure initialization on plugin activation.
 		if ( ! did_action( 'init' ) ) {
-			add_action( 'init', array( $admin_view, 'init' ), 0, 0 ); // run before $store::init()
+			// phpcs:ignore Squiz.PHP.CommentedOutCode
```

Comment:
Is this line required (which commented out code does it refer to)?

```suggestion
```

Reply:
`// run before $store::init().`

Reply:
Interesting. It isn't triggered for me (ie, I don't see an error for that rule, if the `phpcs:ignore` is removed).

Reply:
(Though I would see it if I commented out the full line...) are you invoking phpcs via Composer (are you using the project's PHPCS)?